bedrock/firefox/templates/firefox/hello/index-2016-b.html

```diff
@@ -0,0 +1,22 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/hello/index-2016.html" %}
```

Comment:
This new Hello page is currently behind waffle, and won't be switched on until March 8. Should we be testing the old template instead? If we're switching templates in production mid-test, could this also effect results?